.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.7
```

Comment:
I went with `1.7.x` for now (68d5bc4f04bc55e9241431c618fcef6637dc7918) to avoid upgrading implicitly on unrelated changes.